src/Conversion/ONNXToTorch/NN/Gemm.cpp

```diff
@@ -264,6 +266,8 @@ class ONNXGemmOpToTorchLowering : public OpConversionPattern<ONNXGemmOp> {
     Value transposeBVal = (transB != 0) ? rewriter.create<AtenTOp>(
                                               loc, transposeBType, adaptor.B())
                                         : adaptor.B();
+    setLayerNameAttr(op, transposeAVal.getDefiningOp());
```

Comment:
When several ops need to be created, you've opted to give them all the same `layer_name`.

For RTP generation I don't think this will cause problems as one of these nodes will be used as the representative node for the original operation. @jcamacho1234 I hope you can confirm.

It may be problematic for generating valid visualization files as I believe there is a current assumption that the `layer_name` is unique per visualized operation. If it is problematic, we need to have a look to see if it should be addressed in visualization or by a more refined naming scheme here.

Reply:
> For RTP generation I don't think this will cause problems as one of these nodes will be used as the representative node for the original operation. @jcamacho1234 I hope you can confirm.

As long as this is true, then it should be ok for the RTPGen.